docs/getting_started.md

```diff
@@ -58,12 +58,27 @@ kubectl apply -f ./cluster.yaml -n $(TEST_NAMESPACE)
 </pre>
 
 ### Access workload Cluster
-To access resources on the cluster, you can get the kubeconfig from the secret as following
+To access resources on the cluster, you can get the kubeconfig as following
 <pre>
-kubectl -n ${TEST_NAMESPACE} get secret ${TEST_CLUSTER_NAME}-kubeconfig -o json | jq -r .data.value | base64 --decode > ${TEST_CLUSTER_NAME}.kubeconfig
+clusterctl get kubeconfig ${TEST_CLUSTER_NAME} -n ${TEST_NAMESPACE} > ${TEST_CLUSTER_NAME}.kubeconfig
 kubectl --kubeconfig ./{TEST_CLUSTER_NAME}.kubeconfig get nodes 
 </pre>
 
+### Install CNI on workload Cluster
+
+You must deploy a Container Network Interface (CNI) based Pod network add-on so that your Pods can communicate with each other. Cluster DNS (CoreDNS) will not start up before a network is installed.
+
+Take care that your Pod network must not overlap with any of the host networks: you are likely to see problems if there is any overlap. (If you find a collision between your network plugin's preferred Pod network and some of your host networks, you should think of a suitable CIDR block to use instead, then use it inside the `cluster.yaml` generated by your `clusterctl generate cluster` before applying it)
+
+Several external projects provide Kubernetes Pod networks using CNI, some of which also support [Network Policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
+
+See a list of add-ons that implement the [Kubernetes networking model](https://kubernetes.io/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model). Most common are [Calico](https://www.tigera.io/project-calico/) and [Cilium](https://cilium.io).
+
+You need to follow specific install guide for your selected CNI and you can install only one Pod network per cluster.
+
+Once a Pod network has been installed, you can confirm that it is working by checking that the CoreDNS Pod is Running in the output of kubectl get pods --all-namespaces.
+
```

Comment:
not in pre but I will propose a flavour based CNI install soon

I don't like adding command for product we don't manage because if they change their install method our doc become bad, your view ?